src/main/java/core/basesyntax/strategy/Operations.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.strategy;
+
+public enum Operations {
```

Comment:
Having Operation and Operations enums causing confusion during code reading.
Also enum naming should be in singular.

```suggestion
public enum OperationStrategy {
```